### PR TITLE
Openms idxml

### PR DESCRIPTION
**Pull request by rolivella**
This is for uploading data to test OpenMS idXML module

### CODE REVIEW

**Comment by ggabernet** [comment]
Hi @rolivella , thanks for adding the test data! Could you have a look at the `README.md` and add the data in the corresponding subfolder (e.g. `proteomics`)? Also a short description of where the data comes from

**Comment by rolivella** [comment]
@ggabernet I just added the documentation and pull request

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Can see any changes in the readme :thinking: 
Maybe also it will be nice to rename the file in a more descriptive way.

**Comment by rolivella** [comment]
@JoseEspinosa I already made your requested changes. Could you merge please?

**Review by ggabernet — APPROVED** [review]
Looks good to me know, even though if you have a source where you took the data from, it would be great to also add the reference here

**Comment by ggabernet** [comment]
Hi @rolivella, checking this PR again, we try to keep an orderly folder structure, so that it will be easy to find the test data by everyone.
It seems like this is the first proteomics dataset! 🎉 
I would be therefore great to add it to a new directory called: "proteomics", and a subdirectory with the name of the species that the data comes from. See the other directory structure for `e.g. genomics` for an inspiration.

**Comment by rolivella** [comment]
Hi @ggabernet I think that there's already a folder called "proteomics" here: 

https://github.com/nf-core/test-datasets/tree/modules/data/proteomics

But is not following the structure you mentioned with the species and so on. What should I do?

**Comment by ggabernet** [comment]
In that case you can add a subdirectory under `proteomics` with the species name, and add your test data underneath. So that other proteomics datasets can follow the recommended structure 😄

**Comment by rolivella** [comment]
@ggabernet ok! I'll do it and let you know!

File: README.md [hunk 1]
```diff
@@ -399,6 +399,21 @@ If you cannot find suitable test data on this repository, please contact us on t
...
+  - 'OVEMB150205_14.raw': Thermo RAW mass spectra file.
+
+- database
+  - 'yeast_UPS.fasta': FASTA database for Yeast organism.
+
+- maxquant
+  - 'mqpar.xml': MaxQuant parameters file.
+
+- idxml: 
+  - 'BSA_QC_file.idXML': identifications file for Bovine (BSA) organism coming from an OMSSA search engine.
```
Comment (ggabernet) [inline README.md:415]:
```suggestion

- Bos taurus
  - idxml: 
    - 'BSA_QC_file.idXML': identifications file for Bovine (BSA) organism coming from an OMSSA search engine.
```

**Review by ggabernet — COMMENTED** [review]
Looks good, I would just also add the folder in the readme file as suggested

**Comment by ggabernet** [comment]
Great, looks good to me then! @JoseEspinosa is there any other changes that you wanted there?

**Review by JoseEspinosa — APPROVED** [review]
LGTM @ggabernet ! Very nice 🚀

**Pull request merged**